src/client/Microsoft.Identity.Client/Internal/Logger/MsalLoggerExtensions.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.IO;
+using System.Runtime.CompilerServices;
+using Microsoft.Identity.Client.Internal.Logger;
+using Microsoft.IdentityModel.Logging.Abstractions;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Extension methods for ICoreLogger
+    /// </summary>
+    internal static class MsalLoggerExtensions
+    {
+        public static void Always(this IIdentityLogger logger, string message)
```

Comment:
Adding extension methods here to help avoid the need to change 100s of lines of logger statements in the codebase. these will simply adapt the current log statements to the new LogEntry parameter